### DISCUSSION TITLE
Disable wikilink with rules pragma does not work with transclusion widget

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#6712] [General]
I like to disable wikilink in a tiddler so I use

```
\rules except wikilink
```

This works fine.

If I have a transclusion, then the rules does not work for example create a new tiddler in https://tiddlywiki.com with below contents

```
\rules except wikilink

This ''is''  [[HelloThere]] and https://tiddlywiki.com

and this is also HelloThere

This is transclusion <$transclude tiddler="HelloThere" field=title/>
```

I expect no wikification for camelcase words from transclude output.

**Comment by Jermolene** [comment]
Hi @kookma the rules pragma only affects the parsing of the text within which it appears, and its setting is not inherited by transcluded tiddlers. It's a consequence of how things work internally.

**Comment by kookma** [comment]
Solved by @ericshulman
See https://talk.tiddlywiki.org/t/disable-wikilinks-using-rules-pragma-does-not-work-with-transclude-widget/3546/4
